### PR TITLE
[INDEX-15] Allow trigger retrieval using API

### PR DESCRIPTION
**Pull request by MarekSuchanek**

### CODE REVIEW

File: src/main/java/solutions/fairdata/fdp/index/api/filters/TokenFilter.java [hunk 1]
```diff
@@ -43,16 +43,18 @@
...
     @Autowired
     private TokenService tokenService;
 
     @Override
     protected void doFilterInternal(HttpServletRequest request, HttpServletResponse response, FilterChain filterChain) throws IOException, ServletException {
         String token = getToken(request);
+        System.out.println("TOKEN: " + token);
         if (token != null) {
             Optional<Authentication> auth = tokenService.getAuthentication(token);
+            System.out.println(auth);
```
Comment (kburger) [inline src/main/java/solutions/fairdata/fdp/index/api/filters/TokenFilter.java]: slf4j logger with a debug statement?

File: src/main/java/solutions/fairdata/fdp/index/utils/AdminTriggerUtils.java [hunk 2]
```diff
@@ -34,6 +34,7 @@
 
     public static Event prepareEvent(HttpServletRequest request, Authentication authentication, String clientUrl) {
         var adminTrigger = new AdminTrigger();
+        System.out.println(authentication);
```
Comment (kburger) [inline src/main/java/solutions/fairdata/fdp/index/utils/AdminTriggerUtils.java]: slf4j logger with a debug statement?

**Review by kburger — APPROVED** [review]
👍, minor comments

File: src/main/java/solutions/fairdata/fdp/index/api/filters/TokenFilter.java (hunk 1 above)
Comment (MarekSuchanek) [reply src/main/java/solutions/fairdata/fdp/index/api/filters/TokenFilter.java]: Oops... need to find some checker for those 😅 fixed, thanks

**Pull request merged**